features/FxA-83-sign-in-confirmation-email/README.md

```diff
@@ -0,0 +1,169 @@
+# Sign-in confirmation email
+
+## Problem statement
+
+It is possible for attackers
+armed with a list of known email/password pairs
+to maliciously access users’ Firefox Accounts.
+In such cases,
+attackers would have full access
+to the user’s Sync data,
+including passwords, history, bookmarks and
+the ability to force-install malicious add-ons.
+
+## Outcomes
+
+If an attack occurs,
+we would like fewer accounts to be compromised
+without negatively impacting
+the number of signed-in users
+or our overall FxA engagement rate.
+
+## Hypothesis
+
+If we introduce an additional email confirmation step
+for Sync sign-ins,
+fewer accounts will be compromised.
+We will know this to be true if,
+in the event of an attack,
+the number of successful sign-ins remains constant
+and the count of 104 (unverified account) errors
+from protected endpoints
+increases in line with any wider increase in traffic.
+
+## Assumptions
+
+* The `/account/login`
+  and `/password/change/start` endpoints
+  are responsible for initiating
+  the confirmation process.
+
+* The following endpoints should be protected:
+  * `/certificate/sign`
+  * `/account/keys`
+  * `/account/delete`
```

Comment:
My thoughts were we should protect password change and account delete, but both require a password instead of a sessionToken. While I definitely don't want users to access my Sync data, I'd prefer if they didn't delete my account or change my Sync password as well. Maybe v2?